### PR TITLE
open-cluster-management/backlog#7042 -- deprecation

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

File: manage_applications/channel_sample.adoc [hunk 1]
```diff
@@ -221,7 +221,7 @@ spec:
 apiVersion: v1
 data:
   insecureSkipVerify: "true"
```
Comment (swopebe) [inline manage_applications/channel_sample.adoc:227]: is line 223 the same @rokej ?

File: release_notes/deprecate_remove.adoc [hunk 2]
```diff
@@ -14,7 +14,14 @@ Consider the alternative actions in the _Recommended action_ and details that ar
...
 |===
-|Product or category | Affected item | Status | Version | Recommended action | More details and links
+|Product or category | Affected item | Version | Recommended action | More details and links
+
+| Application management
+| `HelmRepo` channel type
+| 2.2
+| Need recommended action
+| Need details or link if any
+
```
Comment (swopebe) [inline release_notes/deprecate_remove.adoc:24]: I need recommended action and any other details you want for the deprecation topic.

Comment (rokej) [reply release_notes/deprecate_remove.adoc:24]: Until 2.2, you used a configMapRef in channel spec to skip Helm repository server's SSL certificate verification. You would do this if the Helm repository server's SSL certificate is signed by a custom or self-signed CA.

```
apiVersion: v1
kind: Namespace
metadata:
  name: hub-repo
---
apiVersion: apps.open-cluster-management.io/v1
kind: Channel
metadata:
  name: Helm
  namespace: hub-repo
spec:
    pathname: [https://9.21.107.150:8443/helm-repo/charts] # URL points to a valid chart URL.
    configMapRef:  # deprecated for 2.2 release
      name: insecure-skip-verify. # deprecated for 2.2 release
    type: HelmRepo
---
apiVersion: v1
data:
  insecureSkipVerify: "true" # deprecated for 2.2 release
kind: ConfigMap
metadata:
  name: insecure-skip-verify
  namespace: hub-repo
```

This method of specifying `insecureSkipVerify` is being deprecated. 

It is recommended to specify `insecureSkipVerify: true` in channel spec in order to skip Helm repository server's SSL certificate verification.

```
apiVersion: v1
kind: Namespace
metadata:
  name: hub-repo
---
apiVersion: apps.open-cluster-management.io/v1
kind: Channel
metadata:
  name: Helm
  namespace: hub-repo
spec:
    pathname: [https://9.21.107.150:8443/helm-repo/charts] # URL points to a valid chart URL.
    type: HelmRepo
    insecureSkipVerify: true # new way to specify insecure-skip-verify going forward from 2.2 release
```

File: manage_applications/channel_sample.adoc (hunk 1 above)
Comment (rokej) [reply manage_applications/channel_sample.adoc:227]:
```
apiVersion: v1
kind: Namespace
metadata:
  name: hub-repo
---
apiVersion: apps.open-cluster-management.io/v1
kind: Channel
metadata:
  name: Helm
  namespace: hub-repo
spec:
    pathname: [https://9.21.107.150:8443/helm-repo/charts] # URL points to a valid chart URL.
    configMapRef:  # deprecated for 2.2 release
      name: insecure-skip-verify. # deprecated for 2.2 release
    type: HelmRepo
    insecureSkipVerify: true # new way to specify insecure-skip-verify going forward from 2.2 release
---
apiVersion: v1
data:
  insecureSkipVerify: "true" # deprecated for 2.2 release
kind: ConfigMap
metadata:
  name: insecure-skip-verify
  namespace: hub-repo
```

File: release_notes/deprecate_remove.adoc (hunk 2 above)
Comment (swopebe) [reply release_notes/deprecate_remove.adoc:24]: @rokej we can add that to the topic itself, this topic is just a list so I can only add to the table.

Comment (rokej) [reply release_notes/deprecate_remove.adoc:24]: @swopeb OK, you can add this to the topic itself. Then, can you give me an example of what info you would like me to provide for recommended action and any other details?

Comment (swopebe) [reply release_notes/deprecate_remove.adoc:24]: sure, I can take from this in enter the table and you can tell me if it makes sense--I think I have what I need. Will do this update today. Thanks!

**Pull request merged**